rfcs/0098-community-team.md

```diff
@@ -0,0 +1,190 @@
+
+The community team will hold its members accountable to these ideals.
+
+## Code of Conduct and Statement of Values
+[values]: #values
+
+Following the adoption of this RFC, the code of conduct and statement of values will be displayed on a dedicated page on the NixOS website. They will read as follows, unless amended by a future RFC.
+
+The Nix community recognizes that its members come from many walks of life, and that it is natural for us to disagree on some things. Nonetheless, we are all committed to certain values, to allow us to work together and build awesome technologies.
+
+Community members are expected to follow these values in their interactions in all official Nix social spaces, including websites, chat rooms, bug threads, and conferences. We have a community team who will set and enforce social norms, to make that easier for everyone. faCommunity members are expected to follow the community team's requests.
+
+If you have opinions on the community team's work, get involved! It's a rotating team, and we want to develop the leadership potential in all community members.
+
+Our values are as follows.
+
+**Collaboration** — We all want to make the best technologies we can. We work together on our common goals, with open discussion, and without petty infighting or focusing on blame. We are all, always, teaching each other to be better.
+
+**Communication** — When we encounter problems, we focus on solving them, not on fighting each other. When we disagree on technical subjects we follow appropriate processes to escalate them to technical leadership. We try to avoid and de-escalate personal conflicts, but when they are unavoidable we seek mediation.
+
+**Empathy** — We try to see perspectives other than our own, to treat everyone with kindness and empathy, to move past mistakes. When we criticise each other we try to do so graciously. We respect each other's different backgrounds and viewpoints, and we respect the effort everyone puts into the community.
+
+**Humility** — We acknowledge that everybody has something to learn, that nobody is infallible.
+
+**Inclusion** — We work to include everybody, especially people from marginalized backgrounds. We strive to be fair and equitable, and to respect the ways people are different. We reject bigotry in all its forms.
+
+**Accommodation** — When community members face barriers to participation, we accommodate their needs to the best of our ability. This includes, but is not limited to, needs related to disability either physical or mental, to medical issues, to religious observations, to parenting, and to sober lifestyles.
+
+**Transparency** — We recognize that the whole community has a stake in decisions. We strive not to be arbitrary, to disclose when important decisions are made, and to make sure there is clear justification.
```

Comment:
```
**Transparency** — The whole community has a stake in decisions. We strive to disclose important decisions and their rationale.
```
I'm not clear on the "whole community has a stake in decisions" part. The point of this RFC is to define a rotating subset of the community to make particular kinds of decisions. Does stake mean "interest in"? or "is affected by"? If the intention is to make it clear that the entire community needs to be informed:
```
**Transparency** — We strive to disclose important decisions and their rationale to the whole community.
```

Reply:
Sorry, I think I left something out: This isn't just for the behavior of the community team, it's for everyone. There are lots of decisions people might make as community members that affect people. The value is just saying that they should talk about it rather than deciding it without consulting or notifying anyone.

Reply:
> There are lots of decisions people might make as community members that affect people. The value is just saying that they should talk about it rather than deciding it without consulting or notifying anyone.

Do you mean some reasonably large impact? (because discussing everything that somewhat affects people sounds like too much friction for survival of the project)

Reply:
> Do you mean some reasonably large impact? (because discussing everything that somewhat affects people sounds like too much friction for survival of the project)

Yes, exactly. It's up to every community member's judgement to figure out the right balance for the particular situation, and the community team will set social norms around it if it's not working. Things with more impact need to reach a higher bar, but we can't anticipate every possible situation, so we're not trying to. Doing this as a values statement lets us specify what concerns community members should be weighing, without trying to prescribe their conclusions.

Reply:
And I actually object to the notion that in case of Nix* entire community has stake in large-ish decisions. I have zero stake in a large restructuring of Haskell infrastructure, even though it is a large change and impacts a lot of community members.

Reply:
The goal is to give people visibility into the things they care about, and an opportunity to be involved. Of course we don't want to force anyone to get involved with Haskell infrastructure or any other specific area, we only want to make sure that if they want to, they can.

Reply:
Let me try…

**Transparency** — We recognize that people from different parts of the community have a stake in each decision. We strive not to be arbitrary, to disclose when important decisions are made, to provide opportunities for feedback, and to make sure there is clear justification in the end.